graspologic/match/qap.py

```diff
@@ -169,9 +169,9 @@ def quadratic_assignment(A, B, method="faq", options=None):
     return res
 
 
-def _calc_score(A, B, perm):
+def _calc_score(A, B, S, perm):
     # equivalent to objective function but avoids matmul
-    return np.sum(A * B[perm][:, perm])
+    return np.sum(A * B[perm][:, perm]) + np.sum(S[np.arange(len(S)), perm])
```

Comment:
what's more readable, this or `np.trace(S[:, perm])` ?

Reply:
other than this I think it's ready to merge

Reply:
trace is more readable but what you have is fine and possibly faster? totally up to you imo

Reply:
Yeah I made it like this for speed reasons

Reply:
I'll just leave it as is